benchmark/src/DriverBench/ParallelMultiFileExportBench.php

```diff
@@ -74,15 +72,15 @@ public function afterIteration(): void
      * Using a single thread to export multiple files.
      * By executing a single Find command for multiple files, we can reduce the number of roundtrips to the server.
      *
-     * @param array{chunk:int} $params
+     * @param array{chunkSize:int} $params
```

Comment:
Decided to rename this since I got confused as to what `chunk` meant: I thought `chunk: 1` meant a single chunk of files while it actually meant 100 chunks of 1 file each.